### ISSUE TITLE
MassTransferTracker printed in output

### ISSUE THREAD

**Issue by debatric** [TeamCOMPAS/COMPAS#171]
The MassTransferTracker is printed to be "0" during CE mass accretion, even when there is accretion. It is not affecting the pulsar code, since it is taking into account the correct mass accreted, calculating the pulsar parameters correctly and updating the NS mass too. It is only the tracker that is not updated while printing.

**Comment by ilyamandel** [comment]
@debatric -- there is insufficient information to reproduce the bug.  As @SimonStevenson asked, it's impossible to tell whether you are referring to a code variable (if so, which variable is not being updated in what class?), or whether it is printed output (if so, which column / variable in which file)?   Please clarify this so we can either resolve or close if this is no longer an issue.

**Comment by debatric** [comment]
@ilyamandel after having a look at the old files that made me find the bug in the first place. I notice the 8th column of BSE_Pulsar_Evolution.csv file, 'MT_History' is the m_MassTransferTrackerHistory variable. Ordinarily after the formation of the first NS, my dominant channel would have stable MT and then CE, hence  m_MassTransferTrackerHistory being the values 2 and 4, denoting the steps. The bug I saw testing beta was the lack of the first case and hence the said variable was not being updated. However, if I remember correctly the accretion onto NS issue was solved and hence that ought to solve this problem as well.
